ffmpeg/transcoder.c

```diff
@@ -228,8 +232,16 @@ int transcode(struct transcode_thread *h,
     AVFrame *last_frame = NULL;
     av_frame_unref(dframe);
     ret = process_in(ictx, dframe, ipkt);
-    if (ret == AVERROR_EOF) break;
-                            // Bail out on streams that appear to be broken
+    if (ret == AVERROR_EOF) {
+      if(filters_state == FILTERS_NORMAL && ictx->flushing) {
+        // TODO: Here we need to flush the filtergraph and encoders
```

Comment:
Is this what we're doing now? Can we remove the TODO?

Reply:
Fixed